test/ote/utils.go

```
@@ -0,0 +1,319 @@
+	if destDir != "" {
+		args = append(args, "--dest-dir="+destDir)
+	}
+	if imageStream != "" {
+		args = append(args, "--image-stream="+imageStream)
+	}
+	if len(params) > 0 {
+		args = append(args, "--")
+		args = append(args, params...)
+	}
+	fmt.Fprintf(g.GinkgoWriter, "Running: oc %s\n", strings.Join(args, " "))
+	output, err := exec.Command("oc", args...).CombinedOutput()
```

golangci-lint (2.12.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)